src/redprl/tactical.fun

```diff
@@ -60,9 +60,8 @@ struct
 
     fun seq (mt1 : multitactic, (us : Sym.t list, mt2 : multitactic)) : multitactic = fn alpha => fn st =>
       let
-        val beta = Spr.prepend us alpha
-        val (beta', modulus) = Spr.probe (Spr.prepend us beta)
-        val st' = mt1 beta' st
+        val (beta, modulus) = Spr.probe (Spr.prepend us alpha)
```

Comment:
Likewise, I'm just flabbergasted that we didn't notice this before! Really good catch.

Reply:
Agreed! It must be my daily prayer to the god of syntax. Cannot think of any other reason.